### PR TITLE
new ported module: PrintStringIntoString

### PR DESCRIPTION
**Pull request by jessdtate**
No UI or output yet, but should build.

### CODE REVIEW

**Comment by dcwhite** [comment]
@jessdtate No problems given what we discussed, but I'll wait on merging this for two reasons:
* Wait until Travis build problem is fixed, so this can be properly auto-checked;
* I'm going to look at the dynamic port-related code, to see if that is why it's not outputting anything. 
When you tackle the UI, you can just add on commits to this open PR.

**Comment by jessdtate** [comment]
this module should be ready to go.  It doesn't have an algorithm, but we are going to get to that later.

File: src/Modules/Legacy/String/PrintStringIntoString.h [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ DEALINGS IN THE SOFTWARE.
+ */
+
+#ifndef MODULES_STRING_PRINTSTRINGINTOSTRING_H
+#define MODULES_STRING_PRINTSTRINGINTOSTRING_H
+
+#include <Dataflow/Network/Module.h>
+#include <Modules/Math/share.h>
```
Comment (dcwhite) [inline src/Modules/Legacy/String/PrintStringIntoString.h]: @jessdtate Windows build problem (that I mentioned in the lunch): this needs to be the new share.h, i.e. Module/Legacy/String/share.h

**Comment by dcwhite** [comment]
@jessdtate Build is good and a quick check showed it's working. Could you include a simple test network in src/ExampleNets/regression? Once that's in I will merge it.

**Pull request merged**